src/styles/common.css

```diff
@@ -227,6 +228,16 @@ label > .text-field,
   transition: transform ease 0.2s;
 }
 
```

Comment:
I think it will be better if you put padding in a `chip` class and colors in a modifier class

Reply:
Done